fec_eregs/templates/regulations/main-header.html

```diff
@@ -10,33 +10,42 @@
   <div class="masthead">
     <a href="{{ FEC_CMS_URL }}/" title="Home" class="site-title"><span class="u-visually-hidden">Federal Election Commission | United States of America</span></a>
     <ul class="utility-nav list--flat">
-      <li class="utility-nav__item is-disabled">About</li>
+      <li class="utility-nav__item"><a href="{{ FEC_CMS_URL }}/about/">About</a></li>
       <li class="utility-nav__item"><a href="{{ FEC_CMS_URL }}/calendar/">Calendar</a></li>
       <li class="utility-nav__item is-disabled"><span class="js-glossary-toggle glossary__toggle">Glossary</span></li>
     </ul>
   </div>
   <nav class="site-nav js-site-nav">
     <div id="site-menu" class="site-nav__container">
       <ul class="site-nav__panel site-nav__panel--main">
-        <li class="site-nav__item site-nav__item--with-dropdown" data-submenu="data">
-          <a href="{{ FEC_WEB_URL }}/" class="site-nav__link">Campaign finance data</a>
+        <li><h2 class="site-nav__title u-under-lg-only">Menu</h2></li>
+        <li class="site-nav__item u-under-lg-only">
+          <a class="site-nav__link" href="{{ cms_url }}" rel="home">
+            <span class="site-nav__link__title">Home</span>
+          </a>
         </li>
-        <li class="site-nav__item">
-          <a href="{{ FEC_CMS_URL }}/registration-and-reporting/" class="site-nav__link">Registration and reporting</a>
+        <li class="site-nav__item" data-submenu="data">
+          <a href="{{ FEC_WEB_URL }}" class="site-nav__link {% if parent == 'data' %}is-parent{% endif %}">
```

Comment:
You can remove the `if parent` checks and just add `is-parent` to legal resources.